src/sentry/api/endpoints/project_rule_actions.py

```diff
@@ -60,7 +65,58 @@ def post(self, request: Request, project) -> Response:
             project, platform=project.platform, default="javascript", tagged=True
         )
 
+        if features.has(
+            "projects:verbose-test-alert-reporting", project=project, user=request.user
+        ):
+            return self.execute_future_on_test_event(test_event, rule)
+        else:
+            # Old existing behavior to wrap this in a handler which buries exceptions
+            for callback, futures in activate_downstream_actions(rule, test_event).values():
+                safe_execute(callback, test_event, futures)
+            return Response()
```

Comment:
lol i guess we never used this toaster then 🤔  https://github.com/getsentry/sentry/blob/84e40a838212ae8bef95334c0e93169b27c5693c/static/app/views/alerts/rules/issue/index.tsx#L466

Reply:
We do display it when we generate validation messages with this code:
```python
serializer = RuleActionSerializer(
            context={"project": project, "organization": project.organization}, data=request.data
        )

        if not serializer.is_valid():
            raise ValidationError(serializer.errors)
```

Misconfigured slack alerts show this toast message because we validate the slack alerting config earlier in the pipeline than our Jira integrations, though there's not really a good way to do this with Jira's API afaict.